CONTRIBUTING.md

```diff
@@ -0,0 +1,149 @@
+# Contributing Guide
+
+Thank you for deciding to contribute and help us improve Percona Distribution for PostgreSQL documentation!
+
+We welcome contributors from all users and community. By contributing, you agree to the [Percona Community code of conduct](https://github.com/percona/community/blob/main/content/contribute/coc.md).
```

Comment:
Just wondering: if we're pointing to a central CoC that applies to all Percona projects here, is there a point in including a (potentially obsolete) copy of it in this repo?

Reply:
Yes, you are absolutely right. I first included this doc before the link to it in the community repo was provided. Of course, it makes no sense keeping it here, will remove it.